src/components/twilio-voice-basic-call-control/public/index.html

```diff
@@ -1 +1,21 @@
-<!-- TODO: Implement example usage of the Web Component -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Twilio Voice Dialer</title>
+  </head>
+  <body>
+    <h1>Twilio Voice Dialer</h1>
+
+    <twilio-voice-dialer recipient register>
+      <twilio-voice-basic-call-control></twilio-voice-basic-call-control>
```

Comment:
Will a self closing tag work? `<twilio-voice-basic-call-control/>`

Reply:
Sadly Custom elements cannot be self closing 
https://github.com/WICG/webcomponents/issues/624